### DISCUSSION TITLE
Validate array in Form

### DISCUSSION THREAD

**Discussion by fusionconsulting** [livewire/livewire#6928] [1. Help] [answered]
I'm migrating a project from Livewire 2 to Livewire 3 and have a question regarding validating arrays.

In my existing component I have this rule:

'opening_times.*.open'         => 'required|boolean'

How would I accomplish this with the new Form / #[Rule] syntax?

**Comment by Etzahu** [comment, marked as answer]
https://livewire.laravel.com/docs/validation#form-objects

**Reply by fusionconsulting** [reply]
Thanks, I somehow missed that on the first pass. I have since implemented as per the documentation.
